misc/rapiddisk-rootdev/install_initrd.sh

```diff
@@ -38,61 +48,90 @@ is_num()  {
 
 myerror()  {
 
-	ihelp
-	echo "**** $1"
+#	ihelp
+	echo "**** Error: $1 Exiting..."
 	exit 1
 
```

Comment:
I see a space before the end of the function which is fine, as long as we maintain consistency with other functions. I see others that don't adopt this formatting.

Reply:
Now there is an empty line before and after the function and the function body inside `{}`

    #!/bin/bash
    
    test01 () {
    
        echo foo
        echo bar
    
    }
    
    test02 () {
    
        echo foo
        echo bar
    
    }
    
    echo do_something

Reply:
Scroll up to misc/rapiddisk-rootdev/centos/96rapiddisk/module-setup.sh and the check() function. It does not follow this style. All I ask is that all files remain consistent. Not a huge deal.

Reply:
Fixed! Those files were verbatim copies of examples. I did not wrote them by hand. Anyway are ok now.